test/test_torch.py

```diff
@@ -1706,6 +1706,12 @@ def test_add(self):
             expected = torch.zeros(2, 3, device=device).bool()
             self.assertEqual(res, expected)
 
+    def test_bool_sub(self):
+        for device in torch.testing.get_all_device_types():
+            m1 = torch.tensor([True, False, False, True, False, False], dtype=torch.bool, device=device)
+            m2 = torch.tensor([True, True, False, False, False, True], dtype=torch.bool, device=device)
+            self.assertRaises(RuntimeError, lambda: m1 - m2)
```

Comment:
can you, please, add cases 
1) self.assertRaises(RuntimeError, lambda: 1 - m1)
2)  self.assertRaises(RuntimeError, lambda: m1 - 1)